src/main/web/templates/handlebars/main.handlebars

```diff
@@ -12,8 +12,18 @@
         {{#if_any (eq type "bulletin") (eq type "article") (eq type "compendium_landing_page") (eq type "compendium_chapter")}}
             <link rel="canonical" href="{{uri}}" />
         {{/if_any}}
-        <link rel="stylesheet" type="text/css" href="{{#if is_dev}}http://{{location.hostname}}:9000/dist{{else}}//cdn.ons.gov.uk/sixteens/24764b6{{/if}}/css/main.css">
-		{{#if pdf_style}}<link rel="stylesheet" type="text/css" href="{{#if is_dev}}http://localhost:9000/dist{{else}}//cdn.ons.gov.uk/sixteens/24764b6{{/if}}/css/pdf.css">{{/if}}
+
+		<!--[if IE]><![endif]-->
+		<!--[if lte IE 8]>
+		<link rel="stylesheet" href="{{#if is_dev}}http://{{location.hostname}}:9000/dist{{else}}//cdn.ons.gov.uk/sixteens/be61a32{{/if}}/css/old-ie.css"/>
+		<![endif]-->
+        <!--[if IE 9]>
+              <link rel="stylesheet" type="text/css" href="{{#if is_dev}}http://{{location.hostname}}:9000/dist{{else}}//cdn.ons.gov.uk/sixteens/be61a32{{/if}}/css/ie-9.css">
+        <![endif]-->
+		<!--[if gt IE 9]><!-->
```

Comment:
I think this was removed in a recent commit. Could you check if you've pulled from the latest version of Develop and merge that into your branch here please?